include/picongpu/param/speciesAttributes.param

```diff
@@ -147,146 +147,12 @@ namespace picongpu
-     * - files are not shipped with PIConGPU, see documentation for creation and sources
-     * - all paths are either relative to the simOutput folder, or absolute
-     */
-    alias(pressureIonizationStatesFileName);
-
-    /** alias for particle flag, number of atomic states in the atomic state input data,
-     *
-     * see atomicPhysicsAtomicStatesFileName particle flag
-     * - must be equal to the number of atomic states in the atomic state input file
-     */
-    alias(numberAtomicStates);
-
-    /** path to file containing bound-bound transitions input data for atomicPhysics,
-     *
-     * file gives parameter of modelled atomic physics transition for ion species
-     * - must follow PIConGPU specific formatting
-     * - files are not shipped with PIConGPU, see documentation for creation and sources
-     * - all paths are either relative to the simOutput folder, or absolute
-     */
-    alias(boundBoundTransitionsFileName);
-
-    /** path to file containing bound-free transitions input data for atomicPhysics,
-     *
-     * file gives parameter of modelled atomic physics transition for ion species
-     * - must follow PIConGPU specific formatting
-     * - files are not shipped with PIConGPU, see documentation for creation and sources
-     * - all paths are either relative to the simOutput folder, or absolute
-     */
-    alias(boundFreeTransitionsFileName);
-
-    /** path to file containing autonomous transitions input data for atomicPhysics,
-     *
-     * file gives parameter of modelled atomic physics transition for ion species
-     * - must follow PIConGPU specific formatting
-     * - files are not shipped with PIConGPU, see documentation for creation and sources
-     * - all paths are either relative to the simOutput folder, or absolute
-     */
-    alias(autonomousTransitionsFileName);
-
-    //! alias for particle flag, setting electron species to spawn upon atomicPhysics ionization
-    alias(ionizationElectronSpecies);
-
-    /** alias for particle flag, indicates species represents an ion and will participate in atomicPhysics step and
-     * ionization potential depression calculation
-     *
-     * any species participating as an ion in the atomicPhysics step. requires atomic input data, specified via
-     * the particle flags:
-     * - atomicDataType
-     * - chargeStatesFileName
-     * - atomicStatesFileName
-     * - boundBoundTransitionsFileName
-     * - boundFreeTransitionsFileName
-     * - autonomousTransitionsFileName
-     * - numberAtomicStates
-     * - ionizationElectronSpecies
-     *
-     * and the particle attributes:
-     * - atomicStateCollectionIndex
-     * - processClass
-     * - transitionIndex
-     * - binIndex
-     * - accepted
-     */
-    alias(isAtomicPhysicsIon);
+    //! configuration of species for atomicPhysics algorithm, see picongpu/particles/atomicPhysics/ParticleType.hpp for
+    //! details
+    alias(atomicPhysics)
```

Comment:
The CI says that `atomicPhysics` is colliding with the namespace `atomicPhysics`. Unfortunally you need to change one.

Reply:
maybe change all namespaces to `atomicPhy`? This wil be a simple search and replace over the code (2min work).

Reply:
wait, maybe you missed the semicolon only should be `alias(atomicPhysics);`

Reply:
well it was both, fixed